src/Sparrow/TimeValue.cs

```diff
@@ -396,5 +396,13 @@ public static DateTime EnsureMilliseconds(this DateTime date)
 
             return date;
         }
+
+        public static DateTime EnsureUtc(this DateTime date)
+        {
+            if (date.Kind == DateTimeKind.Utc)
```

Comment:
```suggestion
            if (date.Kind != DateTimeKind.Local)
```

Reply:
We do not want to touch 'Unspecified' also.

Should this be
```
if (date.Kind == DateTimeKind.Local)
    return date.ToUniversalTime();

return date;
```
?